internals-js/src/federationSpec.ts

```diff
@@ -195,6 +195,7 @@ export class FederationSpecDefinition extends FeatureDefinition {
 export const FEDERATION_VERSIONS = new FeatureDefinitions<FederationSpecDefinition>(federationIdentity)
   .add(new FederationSpecDefinition(new FeatureVersion(2, 0)))
   .add(new FederationSpecDefinition(new FeatureVersion(2, 1)))
-  .add(new FederationSpecDefinition(new FeatureVersion(2, 2)));
+  .add(new FederationSpecDefinition(new FeatureVersion(2, 2)))
+  .add(new FederationSpecDefinition(new FeatureVersion(2, 3)));
```

Comment:
You may need to rebase your branch on top of whatever branch adds in `2.3` support (presumably one of them does it, not sure which though).

Reply:
Found it, it's the branch in this PR https://github.com/apollographql/federation/pull/2294

Note that it's a branch off of Sylvain's forked repo, so you'l need to add his repo as an upstream so you can pull his branch in.

Reply:
The 2.3 spec was actually added in #2279, but that has been merged now. That said, this is all on the `next` branch, not the `main` one, so that's where this should be rebased.